runtime/oti/j9nonbuilder.h

```diff
@@ -6906,8 +6908,9 @@ typedef struct J9CInterpreterStackFrame {
 	 *
 	 * Stack must be 16-byte aligned.
 	 */
-	U_8 jitFPRs[6 * 16]; /* xmm0-5 128-bit OR xmm0-7 64-bit */
+	U_8 jitFPRs[6 * 64]; /* zmm0-5 512-bit OR xmm0-7 64-bit */
```

Comment:
On Windows x86-64, the upper bits of Y/ZMM6-15 (beyond 128-bits) are also volatile.